tests/utils.py

```
@@ -539,3 +539,4 @@ def create_cirros_vm(
         if wait_running:
             running_vm(vm=vm, wait_for_interfaces=False)
         yield vm
+1
```

Ruff (0.14.10)

Found useless expression. Either assign it to a variable or remove it.

(B018)